modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -184,12 +232,16 @@ I: To create this cluster again in the future, you can run:
 I: To view a list of clusters and their status, run 'rosa list clusters'
 I: Cluster '<cluster_name>' has been created.
 I: Once the cluster is installed you will need to add an Identity Provider before you can login into the cluster. See 'rosa create idp --help' for more information.
-I: To determine when your cluster is Ready, run 'rosa describe cluster -c <cluster_name>'.
```

Comment:
This did not show up for you?
Don't think it should be removed

Reply:
In my testing with ROSA 1.2.8, these two lines are not longer included in the output. They are replaced by more detailed output, followed also by a full cluster description. Instead of replacing the lines in this code block with the full updated output, I opted to replace them with three dots, which signifies that the output continues but has not been included here. The additional output after this point is not a requirement in this documented step.